cyberark-secrets-provider-for-kubernetes/templates/secrets-provider.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: {{ include "conjur.deploy-api" . }}
+kind: Job
+metadata:
+  name: {{ .Release.Name }}
+  namespace: {{ .Release.namespace }}
```

Comment:
I would add an option to add labels or annotations on both the Job definition and the template of the pod.
It's really helpful sometimes for the end user to be able to control certain behaviours of the Job and pod, using labels and annotations. For example, a user could set rules in his/her cluster that pods with certain labels get a certain resource quota.

Reply:
You got it!